pkgs/applications/networking/gossa/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, buildGoModule
+}:
+
+buildGoModule rec {
+  name = "gossa";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "pldubouilh";
+    repo = "gossa";
+    sha256 = "sha256-eMO9aoI+otGQcvBUJtxciQ7yhUidYizLrDjMVchH3qA=";
+    fetchSubmodules = true;
+  };
+
+  vendorHash = null;
+  doCheck = false;
```

Comment:
Why is this disabled? If there's a reason, you should include it in the comment.